docs/examples/traefik-ingress.md

```diff
@@ -1,24 +1,25 @@
-# Installing the Traefik Ingress Controller on k0s
+# Install the Traefik Ingress Controller on k0s
 
-In this tutorial, you'll learn how to configure k0s with the
-[Traefik ingress controller](https://doc.traefik.io/traefik/providers/kubernetes-ingress/),
-a [MetalLB service loadbalancer](https://metallb.universe.tf/),
-and deploy the Traefik Dashboard along with a service example.
-Utilizing the extensible bootstrapping functionality with Helm, 
-it's as simple as adding the right extensions to the `k0s.yaml` file 
-when configuring your cluster.
+You can configure k0s with the [Traefik ingress
+controller](https://doc.traefik.io/traefik/providers/kubernetes-ingress/), a
+[MetalLB service loadbalancer](https://metallb.universe.tf/),
+and deploy the Traefik Dashboard using a service sample. With the extensible
+bootstrapping functionality that Helm provides, this is as simple as adding the
+several extensions to the `k0s.yaml` file  when you configure your cluster.
 
+## 1. Configure k0s.yaml
 
-## Configuring k0s.yaml
+Configure Traefik and MetalLB Helm charts to install during cluster bootstrap
```

Comment:
maybe "to be installed"?

Or:

```suggestion
 "Configure k0s to install Traefik and MetalLB during cluster bootstrapping by adding their Helm charts as extensions in the k0s configuration file (`k0s.yaml`).
```
(and maybe somehow link to some other piece of documentation which tells what this extension business is about)

Reply:
Like the suggested change. Put a link in to the Helm charts topic.